Data/Fibbonaci.go

```diff
@@ -0,0 +1,19 @@
+package Data
+
+func Fibbonaci(size int) []int {
+
+	if size <= 0 {
+		return nil
```

Comment:
There is no need to use `else` after `return` statement.

https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting